src/components/Elixir/AddLiquidity/index.tsx

```diff
@@ -327,251 +326,247 @@ const AddLiquidity: React.FC<AddLiquidityProps> = (props) => {
 
   return (
     <Modal isOpen={isOpen} onDismiss={onClose} overlayBG={theme.modalBG2} closeOnClickOutside={false}>
-      <Root>
```

Comment:
due to removal of `<Root>`, there is issue in scroll. due to this issue I added `<Root>`


https://user-images.githubusercontent.com/6604146/235671535-78b45e1d-7a31-46de-a46c-eda2d88b2200.mov